packages/local_auth/local_auth_ios/lib/local_auth_ios.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:local_auth_platform_interface/local_auth_platform_interface.dart';
+import 'package:local_auth_platform_interface/types/auth_messages.dart';
+import 'package:local_auth_platform_interface/types/auth_options.dart';
+import 'package:local_auth_platform_interface/types/biometric_type.dart';
+
+export 'package:local_auth_ios/types/auth_messages_ios.dart';
+export 'package:local_auth_platform_interface/types/auth_messages.dart';
+export 'package:local_auth_platform_interface/types/auth_options.dart';
+export 'package:local_auth_platform_interface/types/biometric_type.dart';
+
+const MethodChannel _channel =
+    MethodChannel('plugins.flutter.io/local_auth_ios');
+
+/// The implementation of [LocalAuthPlatform] for iOS.
+class LocalAuthIOS extends LocalAuthPlatform {
```

Comment:
Actually, hm. I see that the strings, which are a public class, already use `IOS`. Since renaming that would be a breaking change, we'll need to stick with `IOS` there, in which case I think we should have package-level consistency override the style guide and use `IOS` in this instance too.

So this can stay as-is, but I'll leave all these comments so it's clear that it was a deliberate decision rather than an oversight.

Reply:
Just to bring this conversation with myself full circle, the style guide was actually wrong; `IOS` is correct. We've now fixed it, and will use `IOS` going forward.